resources/qml/device-verification/DeviceVerification.qml

```diff
@@ -0,0 +1,532 @@
+import QtQuick 2.3
+import QtQuick.Controls 2.10
+import QtQuick.Window 2.2
+import QtQuick.Layouts 1.10
+import Qt.labs.settings 1.0
+
+import im.nheko 1.0
+
+ApplicationWindow {
+    property bool sender: true
+	title: stack.currentItem.title
+	id: dialog
+
+	flags: Qt.Dialog
+
+	palette: colors
+
+	Settings {
+		id: settings
+		category: "user"
+		property bool emoji_font_family: true
+	}
+
+	height: stack.implicitHeight
+	width: stack.implicitWidth
+	StackView {
+		id: stack
+		initialItem: sender == true?newVerificationRequest:acceptNewVerificationRequest
+		implicitWidth: currentItem.implicitWidth
+		implicitHeight: currentItem.implicitHeight
+	}
+
+	onClosing: stack.replace(newVerificationRequest)
+
+	property var flow
+	Connections {
+		target: flow
+		onVerificationCanceled: stack.replace(partnerAborted)
+		onTimedout: stack.replace(timedout)
+		onDeviceVerified: stack.replace(verificationSuccess)
+
+		onVerificationRequestAccepted: switch(method) {
+			case DeviceVerificationFlow.Decimal: stack.replace(digitVerification); break;
+			case DeviceVerificationFlow.Emoji: stack.replace(emojiVerification); break;
+		}
+	}
+
+	Component {
+		id: newVerificationRequest
+		Pane {
+			property string title: "Sending Device Verification Request"
+			ColumnLayout {
+				spacing: 16
+				Label {
+					Layout.maximumWidth: 400
+					Layout.fillHeight: true
+					Layout.fillWidth: true
+					wrapMode: Text.Wrap
+					text: "A new device was added."
+
+					verticalAlignment: Text.AlignVCenter
+				}
+
+				Label {
```

Comment:
These don't seem to be using the correct color palette, you may need to set the color manually:

![grafik](https://user-images.githubusercontent.com/16312324/85563762-348a3100-b61d-11ea-8060-b1a63393ec71.png)

Reply:
Yeah that's what I was wondering the theming works fine in my system. Okay then,I'll change it to manually setting the color.